README.md

```diff
@@ -166,6 +166,7 @@ are designated with a ✅, and hosted projects with a 💙.
 - [maplibre-google-streetview](https://github.com/rezw4n/maplibre-google-streetview) - Plugin that integrates Google Street View into any MapLibre map.
 - [maplibre-transition](https://github.com/popkinj/maplibre-transition) - A plugin for smooth transitions between map styles. [demo](https://observablehq.com/d/b9a97acdf712a77b)
 
+
```

Comment:
```suggestion
```

Reply:
Can you please remove this extra space from this PR, sorry for nit-picking...

Reply:
I still see it in the github diff page...

Reply:
<img width="1789" height="608" alt="Screenshot 2025-07-30 at 18 26 35" src="https://github.com/user-attachments/assets/38d11271-e9b2-4cc2-bfe1-aa7c65c39b7c" />